ratpack-core/src/test/groovy/ratpack/http/client/HttpClientSmokeSpec.groovy

```diff
@@ -517,4 +519,73 @@ BAR
     text == "bar"
   }
 
+
+  def "can handle compressed responses"() {
+    given:
+    def stringBuf = new ByteArrayOutputStream()
+    def writer = new OutputStreamWriter(new GZIPOutputStream(stringBuf))
+    writer.write("bar")
+    writer.close()
+
+    otherApp {
+      get("foo") {
+        response.headers.add(CONTENT_ENCODING, "gzip")
```

Comment:
Ok, I actually tried setting the header to do compression which, oddly enough, disables compression.
It worked when I just set ```Accept-Encoding``` on the client side and left the server side alone.